Tools-Override/tests.targets

```diff
@@ -27,8 +28,11 @@
 
   <PropertyGroup>
     <TestRuntimeDir>$(RuntimePath)</TestRuntimeDir>
-    <TestHostExecutable Condition="'$(TestHostExecutable)' == ''">$(TestRuntimeDir)\corerun</TestHostExecutable>
-    <XunitExecutable Condition="'$(XunitExecutable)' == ''">$(TestRuntimeDir)/xunit.console.netcore.exe</XunitExecutable>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' != 'Windows_NT'">$RUNTIME_PATH/</TestRuntimeEnvVar>
+    <TestHostExecutable Condition="'$(TestHostExecutable)' == '' And '$(OS)' == 'Windows_NT'">$(TestRuntimeEnvVar)corerun.exe</TestHostExecutable>
```

Comment:
Not in this PR but you've changed the meaning of this variable, might be worth a rename to something like TestHostExecutablePath